<ui version="4.0" >
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Reports...</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <widget class="QWidget" name="startPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>189</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Please select the report type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboReportSelector" />
       </item>
       <item>
        <widget class="QLabel" name="labelDescription" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignJustify|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButtonCancel" >
           <property name="text" >
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNext" >
           <property name="text" >
            <string>&amp;Next...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
